notes/shelley/DeltaQ.md

```diff
@@ -0,0 +1,86 @@
+# QoS / delta-Q estimation
+
+Peer discovery gives us a pool of peers but no information about link quality.
```

Comment:
Except the first & last times we exchanged UDP packets (& we could if we want maintain more, e.g. round-trip timing).